package/govuk-prototype-kit.config.json

```diff
@@ -1,6 +1,7 @@
 {
   "assets": [
-    "/govuk/assets"
+    "/govuk/assets",
+    "/govuk/all.js.map"
```

Comment:
Thanks @nataliecarey

Would you mind updating the `*.mjs` file instead of this one?
[**./src/govuk-prototype-kit/govuk-prototype-kit.config.mjs**#L30](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk-prototype-kit/govuk-prototype-kit.config.mjs#L30)

We've generated it since this PR

* https://github.com/alphagov/govuk-frontend/pull/2851

@36degrees Will we want to add tests for "expected source maps" like we do in this one?
[**./tasks/gulp/__tests__/after-build-package.test.mjs**](https://github.com/alphagov/govuk-frontend/blob/main/tasks/gulp/__tests__/after-build-package.test.mjs)

For example we also have:

```
/govuk/common.js.map
/govuk/i18n.js.map
```

We can either choose to exclude these, or otherwise ensure we include them?

I'm not too familiar with the Prototype Kit but lots of other tooling has "load maps" options where source maps from other files can be included to supplement stack traces etc

**Gulp source maps** (example)
https://github.com/gulp-sourcemaps/gulp-sourcemaps#load-existing-source-maps

```js
gulp.src('src/**/*.js')
  .pipe(sourcemaps.init({ loadMaps: true }))
```